Endace/Endace.json

```diff
@@ -0,0 +1,922 @@
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "transient": false,
+                            "queryString": "_sourceCategory=suricata | count by src_ip | sort by _count | limit 10",
```

Comment:
why `_sourceCategory` is hard coded every where ?

Reply:
@himanshu219 - What is the recommended practice here? Does this need to be something else so that it's properly replaced with what the user specifies in Sumo at app install time? I thought our scripts in the repo handled this for us?

Is there something @asimonsendace needs to change here?

Reply:
@asimonsendace - I see your email response further below. Typically what our partners do is let the user specific the sourceCategory at app install time, then in the instructions for installing the FERs, they call out that the sourceCategory must match what's set for the app. This is why @himanshu219 is asking.

Reply:
yes no need to change, however the FERs too have hardcoded source category so we just have to document that whatever they provide while installing app should be same in the fer deploying script